plugins/inputs/modbusgw/sample.go

```diff
@@ -0,0 +1,104 @@
+package modbusgw
+
+const sampleConfig = `
+[[inputs.modbusgw]]
```

Comment:
don't include the plugin name in the sample config (it gets included automatically)

Reply:
should i fix that, and also change the name, and re-submit?  Or add that as another PR?  Or you fx it when/after you merge?

Reply:
just make the change in a new commit (same branch) and push it here in addition to the first commit. They'll all get squashed into one later when it gets merged.